lldb/source/Commands/CommandObjectMultiword.cpp

```diff
@@ -429,11 +428,12 @@ llvm::StringRef CommandObjectProxy::GetUnsupportedError() {
   return "command is not implemented";
 }
 
-bool CommandObjectProxy::Execute(const char *args_string,
+void CommandObjectProxy::Execute(const char *args_string,
                                  CommandReturnObject &result) {
   CommandObject *proxy_command = GetProxyCommandObject();
-  if (proxy_command)
-    return proxy_command->Execute(args_string, result);
+  if (proxy_command) {
+    proxy_command->Execute(args_string, result);
```

Comment:
I think you meant to put a `return` in this block, so we don't append an UnsupportedError before returning.